### PR TITLE
XML Sitemap - Add sitemap support for AMP pages

### PR DESCRIPTION
**Pull request by contactashish13**
#37

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 define( 'WP_USE_THEMES', false );
 define( 'WP_TESTS_FORCE_KNOWN_BUGS', true );
 define( 'AIOSEOP_UNIT_TESTING', true );
+define( 'AIOSEOP_UNIT_TESTING_DIR', dirname( __FILE__ ) );
```
Comment (EkoJr) [inline tests/bootstrap.php]: @contactashish13

File: tests/modules/sitemap/test-sitemap.php [hunk 2]
```diff
@@ -106,6 +117,50 @@ public function test_exclude_images() {
...
+	 * Test whether the URLs generated by the AMP plugin are added to the sitemap or not.
+	 *
+	 * @ticket 37 XML Sitemap - Add sitemap support for AMP pages.
+	 */
+	public function test_amp() {
+		$amp = 'amp/amp.php';
+		$file = dirname( dirname( AIOSEOP_UNIT_TESTING_DIR ) ) . '/';
+		
+		if ( ! file_exists( $file . $amp ) ) {
+			$this->markTestSkipped( 'AMP not installed. Skipping.' );
```
Comment (EkoJr) [inline tests/modules/sitemap/test-sitemap.php]: The test was marked to skip

Comment (contactashish13) [reply tests/modules/sitemap/test-sitemap.php]: yes, skipping this only if the AMP plugin is not found.

Comment (EkoJr) [reply tests/modules/sitemap/test-sitemap.php]: I probably should of tagged this here too #1527

**Comment by michaeltorbert** [comment]
@contactashish13 Can you fix the conflicts?
Also, whenever an issue/PR is ready for testing, mark it with the needs testing label.

**Comment by contactashish13** [comment]
done

**Comment by wpsmort** [comment]
@contactashish13 Can you fix the conflicts and assign to me for testing please

**Pull request closed without merging**